### PR TITLE
Change MySQL dependency to SQLite

### PR DESCRIPTION
**Pull request by lionello**

### CODE REVIEW

**Comment by lionello** [comment]
Note: need a MySQL->SQLite migration script!

File: verify.php [hunk 1]
```diff
@@ -32,20 +23,20 @@ function mail_and_die($m)
...
 if ($ok) {
-	mysql_query("UPDATE members.Users SET paid_verified = (SELECT submitted FROM members.Payments WHERE id = $paymentid) + INTERNAL $months MONTH WHERE email = $email2", $link)
-		or mail_and_die('mysql_query UPDATE Users error');
+	$link->exec("UPDATE Users SET paid_verified = (SELECT submitted FROM Payments WHERE id = $paymentid) + INTERNAL $months MONTH WHERE email = '$email2'")
+		or mail_and_die('link->exec UPDATE Users error', __FILE__);
 }
 else {
-	mysql_query("UPDATE members.Users SET paid = paid - INTERVAL $months MONTH WHERE email = $email2", $link)
-		or mail_and_die('mysql_query UPDATE Users error');
+	$link->exec("UPDATE Users SET paid = DATE(paid, '-$months MONTH') WHERE email = '$email2'")
```
Comment (nihaopaul) [inline verify.php:36]: consider sprintf

File: verify.php [hunk 2]
```diff
@@ -32,20 +23,20 @@ function mail_and_die($m)
...
-mysql_query("UPDATE members.Payments SET verified = $ok WHERE id = $paymentid", $link)
-	or mail_and_die('mysql_query UPDATE Payments error');
+$link->exec("UPDATE Payments SET verified = $ok WHERE id = $paymentid")
+	or mail_and_die('link->exec UPDATE Payments error', __FILE__);
 
 if ($ok) {
-	mysql_query("UPDATE members.Users SET paid_verified = (SELECT submitted FROM members.Payments WHERE id = $paymentid) + INTERNAL $months MONTH WHERE email = $email2", $link)
-		or mail_and_die('mysql_query UPDATE Users error');
+	$link->exec("UPDATE Users SET paid_verified = (SELECT submitted FROM Payments WHERE id = $paymentid) + INTERNAL $months MONTH WHERE email = '$email2'")
+		or mail_and_die('link->exec UPDATE Users error', __FILE__);
```
Comment (nihaopaul) [inline verify.php:33]: consider sprintf

**Comment by nihaopaul** [comment]
conflicts on this branch also.

**Comment by lionello** [comment]
Rebased onto master.

Still need script to migrate the current MySQL DB. One way is to use `mysqldump` and manually patch the SQL syntax.

**Pull request merged**